client/src/helpers/index.js

```diff
@@ -0,0 +1,105 @@
+import { isEmpty } from 'lodash';
+/**
+ *
+ * @description Abstracts validation functions
+ * @class Helper
+ */
+class Helper {
+  /**
+   *
+   * @description Ensures Admin priviledges to perform certain operations
+   * @static
+   * @param {any} req
+   * @returns true or false
+   *
+   * @memberOf Helper
+   */
+  static isAdmin(req) {
+    if (req.decoded.data.role !== 'admin') {
+      return false;
+    }
+    return true;
+  }
+
+  /**
+   *
+   * @description Ensures all fields are present when adding or updating book
+   * @static
+   * @param {any} req
+   * @returns true or false
+   *
+   * @memberOf Helper
+   */
+  static isDefined(req) {
+    if (Object.keys(req.body).length < 5) return false;
+    return true;
+  }
+
+  /**
+   *
+   * @description Ensures all inputs are validated
+   * @static
+   * @param {any} req
+   * @returns isValid and errors
+   *
+   * @memberOf Helper
+   */
+  static inputValidation(req) {
+    const errors = {};
+    for (let i = 0; i < 5; i += 1) {
+      const field = Object.values(req.body)[i];
```

Comment:
You probably want to use a `let` here

Reply:
Ok thanks. *Eslint* with *airbnb* style guide which we were asked to use, actually detects it and recommends to change it to `const`